Python/Batch/sample4_job_scheduler.py

```diff
@@ -177,7 +191,12 @@ def execute_sample(global_config, sample_config):
         tasks = batch_client.task.list(job_schedule_id)
```

Comment:
I think this shouldn't list the tasks under the job_schedule, but rather the job_schedule's recent job: https://docs.microsoft.com/en-us/rest/api/batchservice/jobschedule/get#recentjob